include/zephyr/acpi/acpi.h

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
Change the commit title "lib: Add acpica library" or something like that. Change the message accordingly (i.e. remove "driver" word"